Feat/tutorial updates (#16)
runwayml/docs: docs/how-to/docker.md

# Running Models Locally

Here at Runway, we use [Docker](https://www.docker.com/) to help us run our AI models locally. You only need to use Docker, if you'd like to run some of the models locally. Here we discuss why we need to install Docker when you run Runway.

> ![Select Model](assets/images/tutorials/tutorial_docker/docker-runway.png)

Reviewer: ```suggestion

```

## Which models can be run Locally?

> When you browse models, you can see in the model information, if it supports CPU and/or GPU. If the model supports CPU you can run on your local harddrive. Here we see the model Facial Landmarks is able to run locally.

Reviewer: ```suggestion
The model information regarding CPU and GPU support lives inside every model view page. If the model supports CPU and/or GPU will be described in the characteristics table. If the model supports CPU you can run it locally on your computer. For example, we can see that the model called Facial Landmarks is able to run locally:
```

![CPU model](assets/images/how-to/run-locally/model-cpu.png)

## What is Docker?

> [Docker](https://www.docker.com/) is a tool which is designed to run, deploy and create applications, in a container. Containers are a way of making an environment similar to that of your computers Operating System (OS). This allows us to control what is installed within a container for your application, so we have only the things that are needed.

Reviewer: ```suggestion
[Docker](https://www.docker.com/) is a tool designed to run, deploy and create applications that live inside containers. Containers are a way of making closed and specific environments for every application. This allows to control what is installed within a container for your application, so you can have only the things that are needed.
```

> Some of you may wonder if [Docker](https://www.docker.com/) is the same as a virtual machine (VM). Docker differs from a VM, as a VM will include a complete OS. The VM will act and run independently from your computer. But Docker in many ways, has a pick-and-choose kinder attitude and will use your computers resources, so it's less independent and arguably in many ways lighter to use.

Reviewer: ```suggestion
[Docker](https://www.docker.com/) is not the same as a virtual machine (VM). Docker differs from a VM, as a VM will include a complete OS. The VM will act and run independently from your computer. But Docker in many ways, has a pick-and-choose kinder attitude and will use your computer's resources, so it's less independent and arguably in many ways lighter to use.
```
